QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,123 @@
+This document is a declaration of software quality for the `foonathan_memory_vendor` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `foonathan_memory_vendor` Quality Declaration
+
+The package `foonathan_memory_vendor` claims to be in the **Quality Level 4** category.
+
+Below are the rationales, notes, and caveats for this claim, organized by each requirement listed in the [Package Requirements for Quality Level 4 in REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+## Version Policy [1]
+
+### Version Scheme [1.i]
+
+`foonathan_memory_vendor` uses `semver` according to the recommendation for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#versioning).
+
+### Version Stability [1.ii]
+
+`foonathan_memory_vendor` is not yet at a stable version, i.e. `>= 1.0.0`.
+
+### Public API Declaration [1.iii]
+
+As a vendor package, which provides CMake files that configure the foonathan_memory package, there is no public API.
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`foonathan_memory_vendor` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+CMake files that are not in the cmake directory are not installed, and are considered private.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`foonathan_memory_vendor` does not contain any c or c++ code, therefore changes will not affect ABI stability.
+
+## Change Control Process [2]
+
+`foonathan_memory_vendor` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-requirements).
+
+### Change Requests [2.i]
+
+This package requires that all changes occur through a pull request.
+
+### Contributor Origin [2.ii]
+
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+Following the recommended guidelines for ROS Core packages, all pull requests must have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
+
+### Documentation Policy [2.v]
+
+All pull requests must resolve related documentation changes before merging.
+
+## Documentation [3]
+
+### Feature Documentation [3.i]
+
+`foonathan_memory_vendor` does not currently document its features.
+
+### Public API Documentation [3.ii]
+
+`foonathan_memory_vendor` has embedded API documentation, and is available in its source files.
```

Comment:
As in [1.iii], better state there is no API for this, maybe something like:

```suggestion
`foonathan_memory_vendor` does not have an API and therefore does not require API documentation.
```

Reply:
Done.